### PR TITLE
Tidy and Revise v6.4 Reflectometry Release Notes

### PR DESCRIPTION
**Pull request by cailafinn**
**Description of work.**
Rewording and reformatting of the release notes for reflectometry ready for the 6.4 release. 

**To test:**
1. Delete `docs/doctrees` from your build. 
1. Build the docs and check they look right & there are no warnings. 
2. Grammar, spelling, and formatting checked. 
3. Links etc. 

*There is no associated issue.*

*This does not require release notes* because **this would get very recursive if there were.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.4.0/reflectometry.rst [hunk 1]
```diff
@@ -9,23 +9,30 @@ Reflectometry Changes
...
+.. image:: ../../images/ISISReflectometryInterface/experiment_settings_tab.png
+   :align: center
+   :width: 700
+
+- The ``per-angle defaults`` table has been renamed to the ``Settings Lookup`` table to better represent that lookups can now be performed using both the angle and the sample title.
+- The :ref:`Runs tab <refl_runs>` of the ISIS Reflectometry Interface now contains a ``Lookup Index`` column to indicate which row from the ``Settings Lookup`` table was used.
+- Processing is now disabled if there are any errors on the :ref:`Experiment Settings <refl_exp_instrument_settings>` table.
+- Groups on the :ref:`Runs tab <refl_runs>` are now highlighted with a paler green when all runs within them have been processed, but the group has not yet been post-processed.
+- The automated template generation from the :ref:`LRAutoReduction <algm-LRAutoReduction>` algorithm has been removed, as it was never used.
+- Removed the "primary fraction correction" from the :ref:`LRAutoReduction <algm-LRAutoReduction>` algorithm, as this correction is no longer in use.
```
Comment (sf1919) [inline docs/source/release/v6.4.0/reflectometry.rst:23]:
```suggestion
- Removed the ``primary fraction correction`` from the :ref:`LRAutoReduction <algm-LRAutoReduction>` algorithm, as this correction is no longer in use.
```

File: docs/source/release/v6.4.0/reflectometry.rst [hunk 2]
```diff
@@ -9,23 +9,30 @@ Reflectometry Changes
...
+- The ISIS Reflectometry interface now logs a warning if starting live data will use potentially unexpected settings.
+- The Instrument Selector tooltips have been updated to clarify that any changes made using these combo-boxes will apply across all of mantid.
```
Comment (sf1919) [inline docs/source/release/v6.4.0/reflectometry.rst:25]:
```suggestion
- The Instrument Selector tooltips have been updated to clarify that any changes made using these combo-boxes will apply across all of Mantid.
```

File: docs/source/release/v6.4.0/reflectometry.rst [hunk 3]
```diff
@@ -9,23 +9,30 @@ Reflectometry Changes
...
-- Removed the automated template generation from the `LRAutoReduction` algorithm, which was never used.
-- Removed the so-called "primary fraction correction" from the `LRAutoReduction` algorithm. This correction is no longer in use.
-- Renamed the per-angle defaults table to the Settings Lookup table to better represent that lookups can now be done by both angle and by sample title.
-- The Runs Tab of the ISIS Reflectometry Interface now contains a Lookup Index column to indicate which of the rows from the Experiment Settings tab was used.
-- The ISIS Reflectometry interface now log a warning if starting live data will use potentially unexpected settings
-- Updated instrument selector tooltips to clarify that changes to this setting apply across all of mantid.
+- **On the ISIS Reflectometry interface's** :ref:`Experiment Settings tab <refl_exp_instrument_settings>`, **you can now specify default experiment settings on a per-sample basis as well as a per-angle basis. The new "Title" field in the lookup table allows you to specify a regular expression. Any runs with a title that match this expression will use those settings by default.**
```
Comment (sf1919) [inline docs/source/release/v6.4.0/reflectometry.rst:12]:
```suggestion
- **On the ISIS Reflectometry interface's** :ref:`Experiment Settings tab <refl_exp_instrument_settings>`, **you can now specify default experiment settings on a per-sample basis as well as a per-angle basis. The new** ``Title`` **field in the lookup table allows you to specify a regular expression. Any runs with a title that match this expression will use those settings by default.**
```

File: docs/source/release/v6.4.0/reflectometry.rst [hunk 4]
```diff
@@ -9,23 +9,30 @@ Reflectometry Changes
...
-- Workbench will no longer crash if ReflectometryBackgroundSubtraction is run with a group workspace when using the Algorithm dialog.
-- Fix a problem where the default processing instructions could be incorrect. Sensible default values are now used for the INTER linear detector if not specified by the user. Previously, an incorrect pattern was being specified resulting in confusing errors such as a reduced workspace with multiple histograms or error messages about invalid detector IDs or angle correction not being possible.
-- Fix a bug where Mantid could hang when running multiple python algorithms simultaneously, e.g. when running live data and processing a batch at the same time.
+- Loading a batch now correctly re-populates the :ref:`Experiment Settings tab's <refl_exp_instrument_settings>` ``Settings Lookup`` table.
+- Fixed a potential crash when running a live data reduction if the :ref:`SliceViewer <sliceviewer>` is open on the live data workspace.
+- Workbench will no longer crash if :ref:`ReflectometryBackgroundSubtraction <algm-ReflectometryBackgroundSubtraction>` is run from the Algorithm Dialog with a group workspace as the input.
```
Comment (sf1919) [inline docs/source/release/v6.4.0/reflectometry.rst:32]: Not sure about this one but think Group Workspace should probably be capitalised. 

```suggestion
- Workbench will no longer crash if :ref:`ReflectometryBackgroundSubtraction <algm-ReflectometryBackgroundSubtraction>` is run from the Algorithm Dialog with a Group Workspace as the input.
```

**Review by sf1919 — CHANGES_REQUESTED** [review]

**Review by sf1919 — DISMISSED** [review]
I'm very happy with the changes. All the links work, it reads well. It's good to go.

**Comment by martyngigg** [comment]
I think the base branch here should be `release-next`?

**Comment by martyngigg** [comment]
No additional changes have appeared from the branch switch the original review is still valid. I'll approve and merge.

**Review by martyngigg — APPROVED** [review]

**Pull request merged**